### ISSUE TITLE
⌨ Form labels not describing purpose of form control

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#4805]
## Description
When a field label ends with three periods, the `ErrorSummary` component shows three periods followed by a colon in the same sentence. Also, field labels should not end in punctuation and they: 

> need to describe the purpose of the form control.

REF: https://www.w3.org/WAI/tutorials/forms/labels/

### Support form
- I'm looking to...

## Screenshot

![Screen Shot 2023-07-21 at 09 25 23](https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/4d2065b9-bc75-4cd7-a51a-217dde585606)

## Proposed Solution

- ~Halfway solution: For labels that end in three periods or an ellipsis, remove three periods or an ellipsis.~
- Best solution: Rewrite labels so that they describe the purpose of the form control.

Please replace "I'm looking to" with "Reason for contact". The answers should be updated as well to:
- Question
- Bug report
- Feedback

In French, please use:

> Motif du contact
> - Question
> - Signalement de bogue
> - Commentaires  		
> 

## Acceptance Criteria
- [ ] three periods or an ellipsis should not appear at the end of form labels
- [ ] form labels describe the purpose of the form control

**Comment by petertgiles** [comment]
Blocked: need new labels without colons

**Comment by mnigh** [comment]
Copywriters reviewing the copy and could lead to changes to labels.

**Comment by RM-1978** [comment]
@gobyrne, @esizer, (@matt?) and I can set aside some time  take a look through UAT and : 

1) Prepare a list of all the form labels
2) Suggest label updates to:
    - Remove ellipsis 
    - Ensure that input labels are very brief and descriptive

**Comment by gobyrne** [comment]
Ok, I put this as blocked-design until we have a specific list of copy to change.

**Comment by RM-1978** [comment]
Hi @gobyrne, as promised,  here is the specific list of copy to change. The table presents the current copy and our suggested copy updates by code ID. 

[Form Labels_consistency verification_2024-01-16 minus the ones that need Josh and Lauren perspectives.xlsx](https://github.com/GCTC-NTGC/gc-digital-talent/files/13955968/Form.Labels_consistency.verification_2024-01-16.minus.the.ones.that.need.Josh.and.Lauren.perspectives.xlsx)

**Comment by mnigh** [comment]
> [@NienkeBr](https://github.com/NienkeBr) there seem to be only two instances left, would this be something you could rewrite?

@NienkeBr thanks for creating https://github.com/GCTC-NTGC/gc-digital-talent/issues/13283 which addressed one of the two instances. Do you have something in mind for the subject field on the contact form so that this issue can be closed out?
